.editorconfig

```diff
@@ -0,0 +1,18 @@
+root = true
+
+[*]
+end_of_line = lf
+indent_style = space
```

Comment:
```suggestion
indent_style = tab
tab_width = 8
```

Reply:
This is temporary until we merge migrate to only use spaces? 

Also I think you want this only for the c files. I don't think we want this also for shell script and others.

Reply:
> This is temporary until we merge migrate to only use spaces?

Yes.

> I don't think we want this also for shell script and others.

The help files also use tabs.
And, obviously, makefiles **must** use tabs.

Reply:
This should be good. In yaml and markdown files you should use spaces so in that specific case I've put `indent_style = space`.

Reply:
> This is temporary until we merge migrate to only use spaces?
> Also I think you want this only for the c files. I don't think we want this also for shell script and others.

Was there a discussion on this?